frontend/storybook/storybook/stories/TextArea/TextArea.stories.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { VStack, TextArea, FormControl, Text } from 'native-base';
+
+import CenterView from '../CenterView';
+
+storiesOf('TextArea', module)
+  .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
+  .add('Overview', () => {
+    return (
+      <VStack space={4}>
+        <FormControl>
+          <FormControl.Label>
+            <Text variant="body2">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat.
+            </Text>
```

Comment:
Why do we have these? It's not really part of the text area

Reply:
I considered it to be part of the form controls, as in, the label. I felt it was quite relevant to the look of the text area.
I felt it would be a bit redundant to give variants to the FormControl.Label (since it looks different for input fields), which would be the same as the Text variants, so I used a Text component to demonstrate what it looks like.

Reply:
Ah yeah I see. Maybe this is ok for now, but if this style is used more extensively than for text area we should think about adding variants.